### ISSUE TITLE
Window renders incorrectly after computer locks or sleep

### ISSUE THREAD

**Issue by ghost** [keepassxreboot/keepassxc#7111]
This is hard to explain but the KeePassXC window will resize to a window within a window after the computer is woken up from sleep or sometimes just after locking/unlocking the computer. 

It's possible to unlock the database and navigate using the keyboard but the mouse doesn't work reliably as where the pointer is, is not where it will actually "click".

Resizing the window does not fix the issue it simply resizes the window in window as well.

Please see the attached image for clarification.

<img width="874" alt="Capture" src="https://user-images.githubusercontent.com/15910606/140593653-63acf5a6-a765-42d4-b62e-4069d09a40a4.PNG">

**Comment by droidmonkey** [comment]
Which operating system?

**Comment by ghost** [comment]
Sorry forgot to add OS details.

Windows 10 x64 fully updated and running 64bit version of the program as
well. And I'm using an AMD graphics card in case that matters (drivers are
fully updated).

**Comment by droidmonkey** [comment]
Are you on a laptop, external monitor, mixture of 4k and non-4k? This is a Qt bug, but want to narrow down what the unique case is.

**Comment by ghost** [comment]
Desktop with a primary 4k monitor and a secondary WUXGA monitor.

Now that I think about it the different resolution monitors are probably the issue. I know Windows 10 infamously has issues with resizing windows when waking up and when using monitors with different resolutions and that maybe affects Qt weirdly.

**Comment by droidmonkey** [comment]
Yup more than likely the problem

**Comment by JohnLGalt** [comment]
Have a _possible_ fix for this (and other Qt-based apps).  Not sure if I got this from another post in these KPXC support threads or not, but I've found this to fix **all** of the little idiosyncrasies that Qt apps (except FDM) exhibit when using 2 differently sized monitors, particularly when using 2 different resolutions that are integer scalar (in my case, my older monitor is a 1080p using 100% DPI scaling and my newer monitor is 4K, using 150% DPI scaling).  I've tested this thoroughly in Windows 11 stable builds (and the last couple of Insider Preview builds as well) but haven't tried in Windows 10 - but, as it is an environment variable, it should be easy enough to test.  Also, to note, I'm using the pre-release 2.7.0 builds of KPXC, so that may play a factor into why these settings work for me.

I have the following set in my **System variables**:

`QT_AUTO_SCREEN_SCALE_FACTOR` value = 0
`QT_SCREEN_SCALE_FACTORS`  value = 2

Of course, this has worked for me, and obviously for whomever reported wherever they reported it at, but YMMV.

HTH
